### ISSUE TITLE
Unable to launch the app!

### ISSUE THREAD

**Issue by DavidRLTG** [OpenSteamClient/OpenSteamClient#37]
I got this error when running it from my Terminal:
`Initialized thread CallbackThread
Initialized thread LoginThread
Initialized thread ComputerInUse
Relaunching
Initialized thread CallbackThread
Initialized thread LoginThread
Initialized thread ComputerInUse
Failed to copy "/home/davidrltg/.local/share/OpenSteam/linux64/steam": filesystem error: cannot copy file: No such file or directory [/usr/lib/opensteam/opensteam] [/home/davidrltg/.local/share/OpenSteam/linux64/steam]
Setting OpenSteam as active install
Relaunching
Failed to relaunch: 2`
Any ideas?
I'm on Devuan OpenRC (Debian Based).

**Comment by nPHYN1T3** [comment]
I could be speaking out of turn but AFAIK the project is essentially dead so don't use it...(for now).

The symlinking for files is a tad overkill and makes a mess but the current builds and source are non-functioning as the whole project is being re-written in another lang.

**Comment by DavidRLTG** [comment]
Darn. Oh well!
I really do hope this project's not dead, because
Modern Steam is fucking hellish to use, so this could really be a great savior.
Thanks for letting me know!

**Comment by nPHYN1T3** [comment]
Not *dead dead* just dead in the current code base...and yes Steam is a dumpster fire. I'm also waiting impatiently for the rewrite heh. Steam completely stopped working for me about a year and a half ago. I wish I could get a class action going about paid product being stolen via a locked out gateway but that's just normal business these days...scumbags. end le rant.

**Comment by DavidRLTG** [comment]
Holy crap! I'm very sorry, man
That really sounds terrible-
What happened to Valve??? 😭
I'll be careful with Steam then, haha
I hope we get the rewrite then, I wish for you to be happy 👍
Have a great day, dude!

**Comment by nPHYN1T3** [comment]
Money happened to Valve heh. That and laziness. If you saw the recent report about Steam literally running rm -rf /home/$USER you will get a taste of how lazy Valve is with some things. This is where their adoption of GTK making Steam inoperable on machines like my workstation that have several GPU's. (GTK broke on purpose XScreen enumeration way back around GTK 3.1ish.) Steam is just cludging things together. Steam no longer works for most anyone running a DE/WM that isn't Gnome/KDE/XFCE/LXDE. They added a half baked fixed that works "some times" for anyone running tiling WM's or something "uncool." (You know, release Steam on a platform about choice and user agency then cripple it unless the user conforms to the norms...)

**Comment by DavidRLTG** [comment]
Damn :(
What sucks is how they removed the ability to disable webhelper in launch arguments
and the entire fucking UI is a web browser now
STEAM eats so much ram and is so slow
it sucks. I miss when valve made great games that weren't cash grabs and were full of soul, and when they actually cared about their players. oh well.

**Comment by nPHYN1T3** [comment]
Yeah, Steam as a platform resembles the Wii shovelware days. As for the RAM there was a short few months where if I launched Steam normally it would write garbage to my GPU VRAM (which was also drawn to screen) until it was full and nuked the GPU...good times. le sigh.

**Comment by DavidRLTG** [comment]
Darn.. Well, we share the pain, heh
It's nice to talk to you. Thanks for being here I guess, makes me feel less alone, friendo.
_never really got along well with someone in github issues. why are most open source software devs dicks? xd_
Agh, either way, i'm rambling xD
Should I even do this here??
😭

**Comment by Rosentti** [comment]
I should be able to look at this tomorrow. The problem is the package servers delete old packages. 

The solution is replacing the package URL with an internet archive link or updating the package version with oswupdater (which may or may not be broken). 

Though it really is a blessing in disquise, since the C++ version is terribly written (as it's my first ever "real" C++ project) and segfaults all over the place. 

There's no real point in trying to fix the C++ version, it'd be easier to start over completely and look at how the APIs work from the C# versions of the interfaces. (wink wink, if someone wants to do it I'm happy to help with the info I've gathered rewriting things properly)

And off topic:
I loved the old Valve. They really did everything themselves, from VGUI, panorama, vprof, vpc to Source. Now it's mostly just CEF and React. Pretty boring if you ask me. Why not make it an electron app at this point?

**Comment by nPHYN1T3** [comment]
Because electron is hot garbage ;)

**Comment by DavidRLTG** [comment]
We need to go back to the old steam client 😭

**Comment by nPHYN1T3** [comment]
Roll back is a big issue on Steam, lots of games (especially Lin or on Prot) often release updates with breaks or regressions but there is no roll back...only forced forward!

**Comment by DavidRLTG** [comment]
Thanks for fixing!
How do I update? I'm on Debian, more specifically Devuan OpenRC.

**Comment by Rosentti** [comment]
> We need to go back to the old steam client 😭

This should also techically be possible, but going back too far (before the React Library update) causes the client to break due to some version issues with appinfo caches.

**Comment by Rosentti** [comment]
You should be able to build it from source, but I can also trigger a GH actions rebuild.

**Comment by Rosentti** [comment]
[this should be it](https://github.com/OpenSteamClient/OpenSteamClient/actions/runs/7640025864/artifacts/1191594439)

**Comment by DavidRLTG** [comment]
Thank you! i'll try it out as soon as I can.

**Comment by DavidRLTG** [comment]
Hey! I got an error while trying to install it :(
https://imgur.com/E5FRqPL.png

**Comment by nPHYN1T3** [comment]
Text is better for reporting errors but yeah, version isn't an int/float/double bla bla bliggedy heh. You could try extracting the deb and hacking up the version so it's numeric.

**Comment by DavidRLTG** [comment]
NO idea how to do that! :sob:

**Comment by nPHYN1T3** [comment]
Right click, extract grep files for version, change version, recompress (bzip) dpkg -i said deb

**Comment by DavidRLTG** [comment]
Hmm. I dunnoo
I still don't get the instructions anyway 😂
I'll probably wait for the rewrite as well, thank you

**Comment by nPHYN1T3** [comment]
Well the "right click" bit may be an issue if your DE/WM/FM only has deb tools associated with the file type meaning it won't give you normal archive actions (it's just bzip) but you can always use open with select your file archiver or decompress via cli.

**Comment by Rosentti** [comment]
I'll build a new release, which has numeric versions. You can grab it from the tags/releases section.

**Comment by DavidRLTG** [comment]
Thank you!

**Comment by Rosentti** [comment]
Turns out the build failed due to some github shenanigans. I'm just gonna edit the version of the CI deb and upload it here.

**Comment by Rosentti** [comment]
https://github.com/OpenSteamClient/OpenSteamClient/releases/download/v0.0.3-alpha-fixed/fixed.deb
